src/lib/features/project/project-controller.ts

```diff
@@ -327,6 +351,24 @@ export default class ProjectController extends Controller {
             serializeDates(applications),
         );
     }
+
+    async getProjectFlagCreators(
+        req: IAuthRequest,
+        res: Response<ProjectFlagCreatorsSchema>,
+    ): Promise<void> {
+        const { projectId } = req.params;
```

Comment:
Small suggestion, but can we type this in the request body? Right now it's implicit, right? Should be specified both here and in the route description earlier in the file.